core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -140,8 +150,13 @@
     /**
      * Creates a new instance.
      */
-    public DefaultRequestLog(RequestContext ctx) {
```

Comment:
How about keeping the old constructor for users who write unit tests with this class? e.g.

```java
public DefaultRequestLog(RequestContext ctx) {
    this(ctx, ...disabled(), ...disabled());
}
```